src/main.py

```diff
@@ -10,11 +10,23 @@
 # pylint: enable=no-name-in-module
 
 
-def main(argv):
+def main() -> None:
     """
     Program entry point.
     """
-    download_stdlib(argv)
+    arg_parser = argparse.ArgumentParser(
+        description="Generates documentation sites for Enso Standard Library."
+    )
+    arg_parser.add_argument("token", help="GitHub user Personal Access Token.")
+    args = arg_parser.parse_args()
+    run_gen(args.token)
```

Comment:
You're missing a whole host of args. All of the things that are constants should be configurable as arguments. The arguments should just _default_ to those constants. The args should then be threaded through your procedures to where they are needed as arguments.